### PR TITLE
Removes old style in examples and creates model object folder

### LINKED ISSUE #121: Update examples to new package style
Currently, some of the examples under inst/scripts still follow the style from the old package.

### PR DESCRIPTION
**Pull request by martinju**
Summary:

- Rewrote or deleted all examples using old code base. Fixes #121 
- Restructured comparison with python implementation
- Creates folder for model objects under inst
- Moved model objects for testing to test/model_objects

### CODE REVIEW

**Comment by martinju** [comment]
OK, I give up. Something is wrong with the tests, which are actually not changed at all. Everything works locally. Has to be a versioning thing, but I am running out of options from my end with R 3.6.1. @nikolase90 Can you take a look?

**Comment by nikolase90** [comment]
@martinju Haha, sure :+1:

**Comment by nikolase90** [comment]
I guess this explains it: https://blog.revolutionanalytics.com/2019/05/whats-new-in-r-360.html

**Comment by nikolase90** [comment]
@martinju Pull the latest version and check if everything pass when you run `devtools::test()`?

**Comment by nikolase90** [comment]
I generated these files using 3.6.1, so the problem was indeed the sampling that was changed with R 3.6.0.

File: inst/scripts/compare_shap_python.R [hunk 1]
```diff
@@ -13,63 +19,42 @@ x_train <- as.matrix(tail(Boston[, x_var], -6))
...
 colnames(x_test) <- colnames(x_train)
 
-# Fitting a basic xgboost model to the training data
-model <- xgboost(
-  data = x_train,
-  label = y_train,
-  nround = 20
-)
+# Reading the R format version of the xgboost model to avoid crash reading same xgboost model in R and Python
+model <- readRDS("inst/model_objects/xgboost_model_object.rds")
```
Comment (nikolase90) [inline inst/scripts/compare_shap_python.R]:
```suggestion
model <- readRDS(system.file("model_objects", "xgboost_model_object.rds", package = "shapr"))
```

File: inst/scripts/compare_shap_python.R [hunk 2]
```diff
@@ -94,62 +79,16 @@ head(Kshap_largesigma)
...
 # Checking the difference between the methods
 mean(abs(as.matrix(Kshap_indep)-as.matrix(Kshap_largesigma)))
-#[1] 6.752507e-08  # Numerically identical
+#[1] 8.404487e-08  # Numerically identical
 
 
-xgb.save(model=model,fname = "inst/compare_lundberg.xgb.obj") # Need to wait a bit after saving and then loading this in python
 
 #### Running shap from Python ####
+reticulate::py_run_file("inst/scripts/shap_python_script.py")
```
Comment (nikolase90) [inline inst/scripts/compare_shap_python.R]:
```suggestion
reticulate::py_run_file(system.file("scripts", "shap_python_script.py", package = "shapr"))
```

**Pull request merged**